parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1745,6 +1749,12 @@ loop = do
       notImplemented = eval $ Notify NotImplemented
       success = respond Success
 
+      getTypesAndTerms :: Names0 -> [(Name, Reference)]
+      getTypesAndTerms names0 = types <> terms
+        where
+          types = R.toList $ Names3.types0 names0
+          terms = fmap (second Referent.toReference) $ R.toList $ Names.terms names0
+
```

Comment:
Could you move this up into `ListDependenciesI`, just because it's a little specific — also note that `Referent.toReference` doesn't always produce term references (specifically: a `Referent.Con` yields a type `Reference`), so in general this function isn't doing the right thing.  

However, it's not a bug in this case, because the `Reference`s that come out `dependencies` and then go into `names0` are always `Referent.Ref`, so which does produce a term `Reference` using `Referent.toReference`.  The `findHistoricalRefs'` functions don't make this obvious, so I apologize.

But in any case, let's make `getTypesAndTerms` local to `ListDependenciesI` for now? And maybe add a note to this effect ^

Reply:
oops, mentioned the wrong username initially above. @pt2121